src/test/java/cc/altius/FASP/dao/unit/UserDaoImplTest.java

```diff
@@ -0,0 +1,208 @@
+package cc.altius.FASP.dao.unit;
+
+import cc.altius.FASP.dao.impl.UserDaoImpl;
+import cc.altius.FASP.exception.AccessControlFailedException;
+import cc.altius.FASP.model.*;
+import cc.altius.FASP.model.rowMapper.*;
+import cc.altius.FASP.service.AclService;
+import cc.altius.utils.DateUtils;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class UserDaoImplTest {
```

Comment:
I wonder if it would have been better to use an Integration test to test this rather - if we switch from JdbcTemplates or change the SQL in any way, the mocks in the test will need to be re-coded